lib/bb/server/process.py

```diff
@@ -390,12 +390,19 @@ def __init__(self, connection, recv):
         self.connection = connection
         self.recv = recv
 
+        self.reply_wait = 5
```

Comment:
Perhaps this is too short a time for polling and will be a lot of messages. How about sticking to the original 30s for this?

Reply:
A lot of messages in cases where it's waiting for a long time is justified and is maybe a good way to attract attention? Per https://bugzilla.yoctoproject.org/show_bug.cgi?id=12116, 5s timeout itself shouldn't happen too frequently. We've been hitting 10s timeout 50% of time so I'm ok with making it 10s. Is that ok or would you still prefer 30s?

Reply:
10s is fine.